src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -337,14 +337,7 @@ private void EnsureEnoughCapacityBeforeAddingNode()
         /// </summary>
         private void SetCapacity(int capacity)
```

Comment:
Since this is now a one-liner, it could probably be eliminated altogether (since really it's a helper method for other resize routines).